tools/query_breakdown/src/main/java/com/google/bigquery/Main.java

```diff
@@ -17,19 +21,24 @@ public static void main(String[] args) {
       System.out.println("You have not specified the input file");
     }
     if (args[1] != null) {
-      timeLimit = Double.parseDouble(args[1]);
+      errorLimit = Integer.parseInt(args[1]);
     }
     if (args[2] != null) {
       outputFile = args[2];
     }
 
```

Comment:
Please create a work item to use a library such as http://commons.apache.org/proper/commons-cli/ to process flags.

Reply:
Addressed in most recent commit :)